.rspec-sudo

```diff
@@ -0,0 +1,2 @@
+--require spec_helper
+--color
```

Comment:
Is this tag meant to guard against the case where there is not a `travis:travis` `group:user` the local_spec test expects? An alternate pattern could be to guard with an environment variable. So for example add `filter_run_excluding` to spec_helper [like this](https://github.com/puppetlabs/bolt/blob/3955ff37c7bb24911a1799cbb0469107aabd4f88/spec/spec_helper.rb#L41) then add the environment variable only in .travis [like this](https://github.com/puppetlabs/bolt/blob/3955ff37c7bb24911a1799cbb0469107aabd4f88/appveyor.yml#L104)